net/WebSocketHandler.hpp

```diff
@@ -778,16 +778,11 @@ class WebSocketHandler : public ProtocolHandlerInterface
             size_t offset = Util::isValidUtf8((unsigned char*)data, len);
             if (offset < len)
             {
-                static const char *imgModel = "child-000 paste mimetype=image/";
-                static std::regex imgRegex("^child-[0-9a-f]{3} paste mimetype=image/$");
-                bool isImagePaste = std::regex_match(data, data + strlen(imgModel), imgRegex);
-                if (!isImagePaste) {
-                    std::string raw(data, len);
-                    std::cerr << "attempting to send invalid UTF-8 message '" << raw << "' "
-                            << " error at offset " << std::hex << "0x" << offset << std::dec
-                            << " bytes, string: " << Util::dumpHex(raw) << "\n";
-                    assert("invalid utf-8 - check Message::detectType()" && false);
-                }
+                std::string raw(data, len);
+                std::cerr << "attempting to send invalid UTF-8 message '" << raw << "' "
```

Comment:
Logging should be very useful, when one is not debugging. But more important, in an optimized build, the assertion won't fire, and we will be writing to stderr, which may get lost and we wouldn't know that we are hitting this case.

Suggestion: serialize to `std::ostringstream` and then write the contents to `LOG_ERR` (`std::cerr` can we written to in debug build, to help debugging, but I don't see how it could be useful for production).